### PR TITLE
OPSIM-1192: make_rolling_footprint work with default values

### PR DESCRIPTION
**Pull request by yoachim**
`make_rolling_footprint` now works with default kwarg values. Had to shuffle a few things around to avoid circular imports. Removed a strange duplicate `_is_in_ra_range` function (strange ruff never noticed that). Added bare-bones unit test for `make_rolling_footprint`.

### CODE REVIEW

**Comment by rhiannonlynne** [comment]
Could you please paste in some screenshots demonstrating that the rolling footprint is starting up in both uniform and not-uniform mode as expected? And also add screenshots of the footprint leading up to year 4 and year 7.

File: rubin_scheduler/scheduler/utils/utils.py [hunk 1]
```diff
@@ -43,6 +44,16 @@
...
 
 
+def ra_dec_hp_map(nside=None):
+    """
+    Return all the RA,dec points for the centers of a healpix map, in radians.
+    """
+    if nside is None:
+        nside = set_default_nside()
+    ra, dec = _hpid2_ra_dec(nside, np.arange(hp.nside2npix(nside)))
+    return ra, dec
```
Comment (rhiannonlynne) [inline rubin_scheduler/scheduler/utils/utils.py:54]: Why is the original call not just replaced with _hpid2_radec?

**Review by ehneilsen — APPROVED** [review]
Looks fine to me.

**Comment by ehneilsen** [comment]
There've been recent changes to `rubin_scheduler` that required changes to `schedview`. If you update your local `rubin_scheduler` and rebase your changes to `schedview`, it should work and you should be testing with the same versions github is.

**Comment by yoachim** [comment]
> Could you please paste in some screenshots demonstrating that the rolling footprint is starting up in both uniform and not-uniform mode as expected? And also add screenshots of the footprint leading up to year 4 and year 7.

That's not what this PR is addressing.

**Pull request merged**